ethyca/fides
# 2270 - Search field on privacy request page isn't working

Closes #2270 

### Code Changes

* **Privacy Requests** - updated Search textbox placeholder text and disabled spellcheck

### Steps to Confirm

**Admin UI**
1. Open browser tab and navigate to http://localhost:3000
2. Login as fides_test user
3. Click `Privacy requests` top nav menu item
4. Verify that the **Search** textbox placeholder is `Search Request ID`
5. Repeat the Privacy Center steps outlined below
6. Refresh the Admin UI `Privacy requests` page
7. Verify that there are 2 DSRs each with a **NEW** status
8. Type the **Request ID** partial value for one of the DSRs
9. Verify that the corresponding DSR record with the given partial **Request ID** value is displayed

**Privacy Center**
1. Open browser tab and navigate to http://localhost:3001
2. Click the `Access your data` card  
3. Input Name, Email, and Phone values
4. Click the **Continue** button
6. Click the `Erase your data` card
8. Input Name, Email, and Phone values
9. Click the **Continue** button

 


### Pre-Merge Checklist

* Documentation:

### Screen Shot

![Screenshot 2023-02-14 at 5 17 53 PM](https://user-images.githubusercontent.com/68459950/218876032-e70ab06c-0ad6-4c2a-8ed3-0a29ce7d2c1e.png)
